src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.scss

```diff
@@ -21,9 +18,9 @@ $content-width: $gene_image_width;
 }
 
 .proteinSummaryTop {
-  align-items: center;
   display: grid;
-  grid-template-columns: [references] 300px [download] 395px;
+  grid-template-columns: [references] 350px [download] 345px;
+  align-items: flex-start;
```

Comment:
Flex-start is supposed to only apply to flexboxes ([docs](https://drafts.csswg.org/css-align/#positional-values)).